source/documentation/tools.md

```diff
@@ -0,0 +1,247 @@
+
+## Managing your analytical tools
+
+Tools on Analytical Platform include RStudio, JupyterLab and Airflow sandbox. To use these, each user must 'start' their own copy of the software. This gives them the benefit of reserved memory space (compared to the more common shared R Studio Server), and some control over the version of R that is running.
+
+To manage your tools:
+
+1. Go the Analytical Platform [control panel](https://controlpanel.services.alpha.mojanalytics.xyz).
+2. Select the __Analytical tools__ tab.
+
+Use the buttons shown against each tool to manage your copy of the tool:
+
+* "Deploy" - The tool is not yet deployed - this is the initial state. You need to "Deploy" to be able to use the tool for the first time. It sets you up with the latest version and starts it. This may few minutes.
+* "Open" - The tool is either "Idled" (configured but not running) or "Ready" (running). If your RStudio, JupyterLab or Airflow instance is inactive on a Tuesday evening it will be idled. Press "Open" to navigate to the tool in your browser, and if it is not running it will start it (run or "unidle" it). Starting a tool usually takes about 30 seconds, but occasionally will take a few minutes.
+* "Restart" - Often problems with the tool can be solved by restarting the software on the server.
+* "Upgrade" - Another release of the tool is available. Occasionally new versions of R Studio are made available on the Analytical Platform. In this case all users will be given the opportunity to upgrade on the control panel. New versions provide new features and bugfixes to the tool. In addition, there some releases come with improvements to the way RStudio is containerized and integrated with Analytical Platform. You should aim to upgrade when it is offered, although just in case it causes minor incompatibilities with your R code, you should not do it in the days just before you have a critical release of your work. When pressed, the status will change to 'Deploying' and then 'Upgraded'. The __Upgrade__ button will no longer be visible (until another version becomes available).
+
+## Package management
+
+A key element of ensuring analysis is reproducible is maintaining a record of the versions of packages used to produce said analysis.
+
+There are a number of tools available for both R and Python to support package management.
+
+For R:
+
+* **conda** - Recommended for R in Analytical Platform. (Conda is only provided on AP with RStudio images which have R 3.5.1 and later. So if you have the older RStudio with R 3.4 then you'll need to use Packrat instead.)
+* **packrat** - not recommended for R, but remains an option.
+
+For Python:
+
+* **venv** and **pip** - Recommended for Python in Analytical Platform, because it's easier and more reliable than conda.
+* **conda environment** installing packages with just **conda** - can help when you have a package package with a C extension that pip struggles with compiling or installing the binaries
```

Comment:
If the packages they use are a C extension, then they should already be packaged as Python wheels with pre-compiled binaries for supported operating systems. My feeling is we should suggest one, preferably obvious, way to do packaging in Python (and that should be `pip` with a `venv`, because it's widespread and community supported). Perhaps we could just mention conda as an option but not encourage it. It's never worked well for me in the past, whereas `pip` et al has.

Reply:
I've added an explicit message on these to say they are "not recommended" to make it clear.